### PR TITLE
Javascript prerequisites

### PR DESCRIPTION
**Pull request by RobertMrowiec**
- [ ] Added javascript_prerequisites file
- [ ] Updated README with linik to ^ file

PR Dependency: 
https://github.com/Selleo/selleo_best_practices/pull/156

### CODE REVIEW

File: backend_developer/nodejs/javascript_prerequisites.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# JavaScript Prerequisites
+
+## 📦 JavaScript Basics
```
Comment (PoslinskiNet) [inline backend_developer/nodejs/javascript_prerequisites.md:3]: @RobertMrowiec I think we have it here: https://github.com/Selleo/DevPath/blob/master/frontend_developer/01_junior_I.md would be good to extract it to don't repeat yourself.

Comment (PoslinskiNet) [reply backend_developer/nodejs/javascript_prerequisites.md:3]: I also applies to HTTP section

Comment (RobertMrowiec) [reply backend_developer/nodejs/javascript_prerequisites.md]: Updated

Comment (PoslinskiNet) [reply backend_developer/nodejs/javascript_prerequisites.md]: Still this change will break backend node part https://github.com/Selleo/selleo_best_practices/blob/master/service_delivery_paths/backend_developer/nodejs/dev_path.md

Comment (RobertMrowiec) [reply backend_developer/nodejs/javascript_prerequisites.md]: Updated description

**Comment by Rxbsxn** [comment]
@RobertMrowiec fix pr description

**Comment by PoslinskiNet** [comment]
@RobertMrowiec make sure that description is rendered properly next time please. I've updated it.

**Pull request merged**